tx-source/dash_core.php

```diff
@@ -19,6 +19,10 @@
 $Definition['%s user(s) found.'] = '%s user(s) found.';
 $Definition['%s user(s) will be affected by this action.'] = '%s user(s) will be affected by this action.';
 
+$Definition['<0/> will suggest answers on Q&A posts'] = '<0/> will suggest answers on Q&A posts';
+$Definition['<0/> will not suggest answers on Q&A posts'] = '<0/> will not suggest answers on Q&A posts';
+$Definition['<0 /> will not suggest answers on Q&A posts'] = '<0 /> will not suggest answers on Q&A posts';
```

Comment:
Looks like these are duplicates. It looks like the second one is the one we are using. But we mostly use no spaces so maybe the first one is preferable. Maybe we can remove the one with a space later.